src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -29,4 +29,7 @@ public class IntakeConstants {
     public static final double INTAKE_Y = 0.443;
     public static final double INTAKE_SIM_ANGLE_OFFSET = -44.8;
     public static final Translation2d ROOT_POSITION = new Translation2d(INTAKE_X, INTAKE_Y);
+    public static final double kP_SIM = 0;
```

Comment:
Should put a blank line before these constants. Also, you should use here [`Slot0Configs`](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/Slot0Configs.html) directly:
```suggestion
    public static final double PID_GAINS = new Slot0Configs().withKP(0).withKI(0).withKD(0);
```
Also, the gains should probably not be different in the real robot and in the simulator. After all, the simulator is supposed to emulate the real world.

Reply:
done.